native/src/modules/endpoint/sagas/watchFetchCategory.js

```diff
@@ -42,8 +43,10 @@ export function * fetchCategory (dataContainer: DataContainer, action: FetchCate
         call(dataContainer.getResourceCache, city, language)
       ])
 
+      const lastUpdate: Moment | null = yield call(dataContainer.getLastUpdate, city, language)
+
       const push: PushCategoryActionType = {
-        type: 'PUSH_CATEGORY',
+        type: loadCriterion.shouldUpdate(lastUpdate) ? 'REFRESH_CATEGORY' : 'PUSH_CATEGORY',
```

Comment:
Why is this dependent on whether it should update? Isn't this rather dependent on whether it was a user triggered refresh or not?

Reply:
Depends on what behavior we are looking for. This way it would update all open screens if there is new content available, so especially also when the update has been performed automatically, e.g. because of the 24h rule.
If we only check for `forceRefresh === true` this wouldn't be the case and it would only update when you refresh manually.
What do you think?

Reply:
Okay, seems legit. The parameter `forceRefresh` is also taken into account, is it?

Reply:
Yep, `shouldUpdate(...)` reads: `return this._forceUpdate || !lastUpdate ||
      lastUpdate.isBefore(moment.utc().subtract(MAX_CONTENT_AGE, 'hours'))`